client/src/pages/LibrarySearchPage/LibrarySearchPage.js

```diff
@@ -1,18 +1,193 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { artistAll } from '../../actions';
+import axios from 'axios';
+
+import ReactTable from 'react-table';
 
-import Button from '../../components/Button';
 import Card, { CardBody } from '../../components/Card';
 import Dropdown, { DropdownItem } from '../../components/Dropdown';
 import Form from '../../components/Form';
 import Input from '../../components/Input';
-import LargeText from '../../components/LargeText';
-import TableArtist from '../../components/TableArtist';
 
 class LibrarySearchPage extends Component {
```

Comment:
This is all straight React rather than Redux-y, I could use feedback on where it makes sense to share the state with the whole app and where it makes sense to keep it contained within the component. I'll also read up on that a bit.